shared/src/common/enums/validation/eam-group/eam-group-validation-message.ts

```diff
@@ -7,6 +7,8 @@ const EAMGroupValidationMessage = {
   NAME_REGEX: 'Group name can contain latin letters, digits, hyphen or dot)',
   PERMISSION_SELECTED_MIN:
     'User can not create a group without any permissions for it',
+  WORKERS_SELECTED_MIN:
+    'User can not create a group without any workers for it',
```

Comment:
Maybe separate messages?
```suggestion
  WORKERS_CREATED_MIN:
    'User can not create a group without any workers for it',
  WORKERS_SELECTED_MIN:
    'Group must have workers',
```